docs/one-click-integrations/github-integration.md

```diff
@@ -139,6 +139,58 @@ The table below lists the data that the GitHub integration collects from your Gi
 
 <sup><span id="deployment-teams">2</span></sup>: Adding a new team or changing the composition of a team on GitHub only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted teams on GitHub will continue to show in Pulse.
 
+## Which permissions does Pulse need from GitHub? {: id="gh-permissions"}
+
+Pulse requests only the necessary permissions from GitHub to collect your organization changes and deployment data and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
+
+<table>
+  <colgroup>
+    <col width="20%"/>
+    <col width="20%"/>
+    <col width="60%"/>
+  </colgroup>
+  <thead>
+    <tr>
+      <th>Scope</th>
+      <th>Permissions</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td colspan="3"><strong>Repository permissions:</strong></td>
+    </tr>
+    <tr>
+      <td>Pull requests</td>
+      <td>Read</td>
+      <td>Pulse retrieves pull request information to calculate several of the metrics presented on the dashboards. <a href="#collected-data">See the details here.</a></td>
+    </tr>
+    <tr>
+      <td>Contents</td>
+      <td>Read</td>
+      <td>Pulse retrieves tag information to support semantic versioning. For this, Pulse GitHub App requires read permission on contents of files and directories, as it's the same permission that applies to tags.</td>
```

Comment:
```suggestion
      <td>Pulse retrieves tag information to support semantic versioning. For this, Pulse GitHub App requires read permission on files and directories, as it's the same permission that applies to tags.</td>
```
`contents of` sounds redundant here. Read permissions already imply access to content.

Reply:
I wrote that way because [Contents](https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28) is in fact the GitHub object we are referring to, and also to match the term in the first column.

Do you think that it will have the same technical meaning if I remove the "contents of" as suggested, @nicklem @prcr ?

Reply:
Hmm, I see. Then my suggestion is to use the same wording as GitHub does:

> [...] requires read permission on **the** contents of files and directories

![image](https://user-images.githubusercontent.com/60105800/205896380-8fa63c1b-9067-4627-9876-753cd4a269a6.png)

Reply:
Thanks! Updated.